src/grisp_connect_logger_bin.erl

```diff
@@ -147,6 +150,27 @@ handle_info(_, _, State) ->
 terminate(_Name, _Reason, _State) ->
     ok.
 
+% %--- Log Filtering Functions ---------------------------------------------------
+
+filter_out(LogEvent = #{meta := Meta}, Mfa = {M, F, A})
+  when is_atom(M), is_atom(F), is_integer(A) ->
+    case Meta of
+        #{mfa := Mfa} -> stop;
+        _ -> LogEvent
+    end;
+filter_out(LogEvent = #{meta := Meta}, {M, F})
+  when is_atom(M), is_atom(F) ->
+    case Meta of
+        #{mfa := {M, F, _}} -> stop;
+        _ -> LogEvent
+    end;
+filter_out(LogEvent = #{meta := Meta}, M)
+  when is_atom(M) ->
+    case Meta of
+        #{mfa := {M, _, _}} -> stop;
+        _ -> LogEvent
+    end.
+
```

Comment:
I do not like filtering logs by default, the user should at least know about it.
What if we add setting that is hide/stop so by default we "hide/replace" supervisor progress report logs with
"hidden supervisor report, modify config to enable" ?

Reply:
This function do not filter by itself, it has to be used in the configuration as it is there: https://github.com/grisp/grisp_connect/blob/4456c9d57af9ca95ca110a57e107c8241ea7eb0b/src/grisp_connect.app.src#L38

Reply:
We could remove it when we fix the crash....

Reply:
Yes but since it is in the app.src it would enabled by default on every release, right?

Reply:
Would you rather document that this would be needed when enabling info logging other wise it would crash ?